west.yml

```diff
@@ -110,7 +110,7 @@ manifest:
     - name: hostap
       repo-path: sdk-hostap
       path: modules/lib/hostap
-      revision: 8e2e758b0296e0f6e8a2261884dca15da37d96a8
+      revision: pull/108/head
```

Comment:
Update to `372c30e40a977c1dbfd6b559eb1e22d447c21e3e`

Reply:
Done